.github/workflows/vue_deploy.yml

```diff
@@ -5,22 +5,22 @@ on:
   pull_request:
     types: [closed]
     branches:
-      - dev
+      - main
 
 jobs:
   aws-deploy:
     runs-on: ubuntu-latest
     #For google auth
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         # may need to change to version 3
```

Comment:
no need to keep this comment and the one on line 21 anymore

Reply:
Updated. Thanks for the feedback.